packages/cloudflare/test/integrations/fetch.test.ts

```diff
@@ -101,8 +101,8 @@ describe('WinterCGFetch instrumentation', () => {
     expect(fetchInstrumentationHandlerCallback).toBeDefined();
 
     const startHandlerData: HandlerDataFetch = {
-      fetchData: { url: 'https://dsn.ingest.sentry.io/1337', method: 'POST' },
-      args: ['https://dsn.ingest.sentry.io/1337'],
+      fetchData: { url: 'https://dsn.ingest.sentry.io/1337?sentry_key=123', method: 'POST' },
+      args: ['https://dsn.ingest.sentry.io/1337?sentry_key=123'],
```

Comment:
this perfectly illustrates the consequences of this change. perviously, this URL was considered a "Sentry request", now we have to add the parameter. This is likely just because the test was "incomplete" before but it does open us up to a few false negatives. Thoughts anyone?

We can further refine the URL matching logic as we want

Reply:
sounds reasonable to me!